### PR TITLE
demo/example widget preview feature

### PR DESCRIPTION
**Pull request by Cthomp7**
- Added preview function for example dialogue
- Fixed standard JS styling (some additional semicolons elsewhere & extra return)

### CODE REVIEW

File: www/widgets/DemoExampleMaker.js [hunk 1]
```diff
@@ -209,6 +213,10 @@ class DemoExampleMaker extends Widget {
 
   // ------------------------------
 
+  _previewStep (step) {
+    window.convo = new Convo({ content: this.$('[name="dem-s-text"]').value })
+  }
```
Comment (nbriz) [inline www/widgets/DemoExampleMaker.js:218]: i'd also add a conditional here to see if there are any "line numbers" specified to highlight, if so we should highlight those using `NNE.spotlight([numbers])` if not we should reset to no spotlight `NNE.spotlight(null)` 

in netnet the global `NNE` variable represents the main instance of the [netitor](https://github.com/netizenorg/netitor) ( "spotlight" is one of it's methods, u can check out all the [other methods in the netitor docs here](https://github.com/netizenorg/netitor#methods)).

**Review by nbriz — CHANGES_REQUESTED** [review]
hey @Cthomp7 this is great, one tiny suggestion (see below) after that this is ready to merge

File: www/widgets/DemoExampleMaker.js [hunk 2]
```diff
@@ -209,6 +213,14 @@ class DemoExampleMaker extends Widget {
...
   // ------------------------------
 
+  _previewStep (step) {
+    window.convo = new Convo({ content: this.$('[name="dem-s-text"]').value })
+    if (this.$('[name="dem-s-focus"]').value.length > 0) {
+      const lines = this.$('[name="dem-s-focus"]').value.match(/\d+/g).map(Number)
+      NNE.spotlight([...lines])
+    }
+  }
+
```
Comment (nbriz) [inline www/widgets/DemoExampleMaker.js:223]: looks great! just missing the `else` on line `221`
```js
else NNE.spotlight(null)
```

**Comment by Cthomp7** [comment]
Added the `else` statement on line `221`

**Comment by nbriz** [comment]
looks great! merging now

**Pull request merged**